README.md

```diff
@@ -97,6 +97,18 @@ git clone https://github.com/seladb/PcapPlusPlus.git
 
 Follow the build instructions according to your platform in the [Build From Source](https://pcapplusplus.github.io/docs/install#build-from-source) page in PcapPlusPlus web-site.
 
+**Using CMake**
+
+```sh
+# build and install using compiler of your choice (for example, clang here)
+CC=clang CXX=clang++ cmake -Bbuild && cmake --build build -j$(nproc) && cmake --install build --prefix /prefix/path
+```
+
+```sh
+# uninstall
+cmake --build build --target uninstall
+```
```

Comment:
This addition is not needed, we already have documentation of how to build with CMake: https://pcapplusplus.github.io/docs/next/install

It's currently on the "Next" release because we don't yet have an official release with CMake, but we'll have it soon hopefully :)

Reply:
Got it, I will update the docs which now removes PCPPP_INSTALL option

Reply:
Thanks! Here is the docs repo: https://github.com/PcapPlusPlus/pcapplusplus.github.io

Specifically, you probably want to update everything under `install`:
https://github.com/PcapPlusPlus/pcapplusplus.github.io/tree/master/docs/install

Reply:
It's not required as I have removed these commits.